symfit/core/minimizers.py

```diff
@@ -511,6 +501,11 @@ def scipy_constraints(self, constraints):
         cons = tuple(cons)
         return cons
 
+    def _pack_output(self, ans):
+        fit_result = super(ScipyConstrainedMinimize, self)._pack_output(ans)
+        fit_result.constraints = self.constraints
```

Comment:
No. Bad. Shouldn't those be part of the Model anyway? Let FitResult take it from there.

Reply:
constraints are not a part of the model, but of `Fit` and/or the minimizer. So I don't see why this is particularly bad? Python is a consenting adults language? :)

Reply:
Shouldn't constraints be part of the Model? (I feel like we've had that discussion before).
Either way, I also don't immediately see a better way of solving htis.

Reply:
No they aren't, but any BaseModel can be used as a constraint now.